### PR TITLE
Add instructions for disabling top-level imports

### PR DESCRIPTION
**Pull request by kant2002**
Based on comment https://github.com/dotnet/runtime/issues/55855#issuecomment-883213093

### CODE REVIEW

**Review by tdykstra — CHANGES_REQUESTED** [review]
Thanks for proposing this change! But I'd prefer to wait until #25250  is merged so we can link to msbuild-props.md#disableimplicitnamespaceimports.

**Comment by RussKie** [comment]
Related https://github.com/dotnet/docs/issues/25436

**Comment by BillWagner** [comment]
ping @tdykstra It looks like #25250 has been merged. Should this be merged now?

File: docs/core/tutorials/top-level-templates.md [hunk 1]
```diff
@@ -57,3 +57,5 @@ These two features simplify the code that makes up your application. To follow a
 If you'd prefer to use the older format, you can copy the code from the second example in this article, and continue the tutorial as before.
 
 You can learn more about top-level statements in the tutorial exploration on [top level statements](../../csharp/whats-new/tutorials/top-level-statements.md).
+
+If you want to opt-out of that behaviour and manually control all namespaces in your project. Add `<DisableImplicitNamespaceImports>true</DisableImplicitNamespaceImports>` in the project file.
```
Comment (tdykstra) [inline docs/core/tutorials/top-level-templates.md:61]:
```suggestion
If you want to opt-out of that behavior and manually control all namespaces in your project, add [`<DisableImplicitNamespaceImports>true</DisableImplicitNamespaceImports>`](../project-sdk/msbuild-props.md#disableimplicitnamespaceimports) in the project file.
```

**Comment by tdykstra** [comment]
> Should this be merged now?

Yes, I added the link.

**Review by tdykstra — APPROVED** [review]

**Pull request merged**